src/main/java/teammates/ui/webapi/endpoints/WebPageEndpoints.java

```diff
@@ -0,0 +1,31 @@
+package teammates.ui.webapi.endpoints;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import teammates.common.util.Const.WebPageURIs;
+
+/**
+ * API endpoints for web pages.
+ */
+public enum WebPageEndpoints {
```

Comment:
Actually could I ask what this auto-sync means? I was trying to follow Wee Han's ResourceEndpoints format because previous i was hardcoding the raw string which felt a little wrong

Reply:
@tiuweehan You might want to explain?

Reply:
The URIs for navigation should stay in the frontend - only the URIs for API calls should be synced in both frontend and backend